roles/config-vnc-server/tasks/vnc-server.yml

```diff
@@ -21,7 +21,7 @@
   register: passwd_info 
 
 - name: "Set a default vnc password (use 'vncpasswd' to change)"
```

Comment:
Maybe just remove the `default` and the statement in the parenthesis since now this isn't just a "default" anymore